cape/cape_result.py

```diff
@@ -1106,10 +1106,10 @@ def convert_sysmon_network(
                     if not is_tag_safelisted(text, ["network.dynamic.domain"], safelist):
                         dns_query["request"] = text
                 elif name == "QueryResults":
-                    ip = search(IP_REGEX, text)
-                    if ip:
-                        ip = ip.group(0)
-                        dns_query["answers"].append({"data": ip, "type": "A"})
+                    ip = findall(IP_REGEX, text)
+                    for item in ip:
+                        if re_match(IPV4_REGEX, item):
```

Comment:
Is there a reason to search using IP_REGEX and validate with IPV4_REGEX instead of searching with IPV4_REGEX to begin with?

Reply:
addressed https://github.com/CybercentreCanada/assemblyline-service-cape/pull/52/commits/2822f26b178030dca864e35b21c122a6fdedb6df